src/components/elements/Inputs/FileInput/FilePreviewCard.tsx

```diff
@@ -2,65 +2,106 @@ import { useT } from "@transifex/react";
 import classNames from "classnames";
 import Lottie from "lottie-react";
 import { Else, If, Then, When } from "react-if";
+import { twMerge as tw } from "tailwind-merge";
 
 import SpinnerLottie from "@/assets/animations/spinner.json";
 import IconButton from "@/components/elements/IconButton/IconButton";
 import { FileCardContent } from "@/components/elements/Inputs/FileInput/FileCardContent";
-import { IconNames } from "@/components/extensive/Icon/Icon";
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
 import { useFileSize } from "@/hooks/useFileSize";
 import { UploadedFile } from "@/types/common";
 
+import Text from "../../Text/Text";
+import { FilePreviewCardVariant, VARIANT_FILE_PREVIEW_CARD_DEFAULT } from "./FileInputVariants";
+
 export interface FilePreviewCardProps {
   file: Partial<UploadedFile>;
   className?: string;
+  variant?: FilePreviewCardVariant;
   showPrivateCheckbox?: boolean;
   onDelete?: (file: Partial<UploadedFile>) => void;
   onPrivateChange?: (file: Partial<UploadedFile>, checked: boolean) => void;
 }
 
-const FilePreviewCard = ({ file, className, showPrivateCheckbox, onDelete, onPrivateChange }: FilePreviewCardProps) => {
+const FilePreviewCard = ({
+  file,
+  className,
+  showPrivateCheckbox,
+  onDelete,
+  onPrivateChange,
+  variant = VARIANT_FILE_PREVIEW_CARD_DEFAULT
+}: FilePreviewCardProps) => {
   const handlePrivateChange = (checked: boolean) => {
     if (!file) return;
 
     onPrivateChange?.(file, checked);
   };
+  const image = { isVerified: true };
```

Comment:
This value never appears to change. Is this unfinished functionality?

Reply:
yeah, it was for the image Status
![image](https://github.com/wri/wri-terramatch-website/assets/139272044/c00edc8b-696e-4e12-893d-5593672b8632)
const image = { isVerified: true }; was deleted and i add a fileStatus prop to controll that it is a boolean for now, until we need more states